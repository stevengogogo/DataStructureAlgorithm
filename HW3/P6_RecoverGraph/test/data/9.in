4
2 3 4
2 4 3
3 1 4 2
3 2 3 1