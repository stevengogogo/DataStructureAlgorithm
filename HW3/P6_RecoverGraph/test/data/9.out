Yes
2 4
1 3
3 4
1 4
2 3